sympy/solvers/inequalities.py

```diff
@@ -317,10 +317,9 @@ def _bottom_up_scan(expr):
                     exprs = args
         elif expr.is_Pow:
             n = expr.exp
-
-            if not n.is_Integer or n < 0:
-                raise ValueError(
-                    "only non-negative integer powers are allowed")
+            
+            if not n.is_Integer:
+                raise ValueError("Only Integer Powers are allowed on Abs.")
```

Comment:
my fault here: change "on" to "with"

Also, please only capitalize Only (start of sentence) and Abs (SymPy function name).